### PR TITLE
PropertyDDS general bugs fixes and performance improvements

### PR DESCRIPTION
**Pull request by nedalhy**
This PR includes few performance and bug fixes commits:

- Improved performance of setValues for arrays.
- Export String Property from property-properties.
- Avoid redundant rebases in propertyTree.
- Increased chunk size.
- Incremental update of the tipView.
- Fixed a bug that caused pruning to fail for a commit with an empty remoteHead

### CODE REVIEW

**Comment by nedalhy** [comment]
@ruiterr PTAL

File: server/tinylicious/config.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     },
     "alfred": {
         "maxMessageSize": "16KB",
+        "restJsonSize": "50mb",
```
Comment (nedalhy) [inline server/tinylicious/config.json:19]: @DLehenbauer is it okay to keep this change?

File: experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts [hunk 2]
```diff
@@ -678,10 +684,14 @@ export class SharedPropertyTree extends SharedObject {
 			applyAfterMetaInformation: pendingChangesRebaseMetaInformation,
 		});
 
-		// Update the the tip view
-		this.tipView = _.cloneDeep(this.remoteTipView);
-		const changeSet = new ChangeSet(this.tipView);
-		changeSet.applyChangeSet(accumulatedChanges);
+		// Udate the the tip view
```
Comment (nedalhy) [inline experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts:687]: typos

File: experimental/PropertyDDS/packages/property-changeset/src/rebase.js [hunk 3]
```diff
@@ -127,32 +129,35 @@ function rebaseToRemoteChanges(change, getUnrebasedChange, getRebasedChanges, is
...
-          changeset = _.cloneDeep(changesOnOtherLocalBranch[0].changeSet);
-          rebaseBaseChangeSet._rebaseChangeSet(changeset, conflicts, { applyAfterMetaInformation });
-
-          // This is disabled for performance reasons. Only used during debugging
-          // assert(_.isEqual(changeset,this.remoteChanges[i].changeSet),
-          //                 "Failed Rebase in rebaseToRemoteChanges");
-          rebaseBaseChangeSet = invertedChange;
-          changesOnOtherLocalBranch.shift();
-        }
+      if (!_.isEqual(changesOnOtherLocalBranch.map((x) => x.guid),
```
Comment (nedalhy) [inline experimental/PropertyDDS/packages/property-changeset/src/rebase.js:132]: Rename parameter `x` to a meaningful name. E.g. `change`

**Comment by nedalhy** [comment]
@DLehenbauer could you review the changes? If it LGTY, then we can merge it. 
I created this PR on behalf of @ruiterr so I already reviewed the changes and it LGTM.

**Comment by nedalhy** [comment]
@DLehenbauer Is it ready to be merged ?

**Review by DLehenbauer — APPROVED** [review]

**Pull request merged**